src/style.css

```
@@ -1,3 +1,83 @@
-body {
+html {
+color: #000;
+font-style: normal;
+font-weight: 500;
+line-height: normal;
+text-transform: uppercase;
+background: gray;
+font-family: Roboto, sans-serif;
+}
+
+body, ul, li {
```

li doesn't have a margin
```suggestion
body, ul {
```